lib/src/presentation/pages/categories/categories_screen_body.dart

```diff
@@ -37,18 +38,22 @@ class _CategoriesScreenBodyState extends State<CategoriesScreenBody> {
                   children: [
                     Expanded(
                       child: TextField(
+                        onTap: () => Navigator.pushNamed(context,PageRouteName.search),
```

Comment:
the navigation should be action also bassing form the view to cubit because you could have a logic between the press on the button and the navigation